app/src/main/java/com/google/samples/apps/nowinandroid/ui/NiaAppState.kt

```diff
@@ -97,6 +105,37 @@ class NiaAppState(
             else -> null
         }
 
+    val backStackEntries: List<NavBackStackEntry>
+        @Composable get() {
+            // TODO: Read backStack directly from the navController when b/295553995 is resolved.
```

Comment:
Nit: Can this be the public issue link? https://issuetracker.google.com/issues/295553995